pages/post/detail/[post_id].vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div>
+
+    <UContainer v-if="post">
+      <div>
+        <p class="font-semibold inline-block w-20">Id:</p>
```

Comment:
Avoid fixing widths statically, unless needed. And even if needed, it should be min- max-
Use flex properties.